Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListCollectionViewCell.swift

```diff
@@ -5,4 +5,36 @@
 import Foundation
 import UIKit
 
-open class ChatChannelListCollectionViewCell: UICollectionViewCell {}
+open class ChatChannelListCollectionViewCell<ExtraData: UIExtraDataTypes>: UICollectionViewCell {
+    
+    // MARK: - Properties
+    
+    var uiConfig: UIConfig<ExtraData> = .default
+    
+    public private(set) lazy var channelView: ChatChannelView<ExtraData> = {
+        let view = uiConfig.channelList.channelView.init()
```

Comment:
we should pass the config down to the channel view

Reply:
fixed in 105094bf5e1ad5b8196f48f8e76edb87adc31b65